specs/engage-digital_openapi3.yaml

```diff
@@ -7388,6 +7404,12 @@ components:
         updated_at:
           format: date-time
           type: string
+        auto_complete_category_ids:
+          items:
+            type: string
+          type: array
+        task_auto_complete_seconds:
+          type: integer

```

Comment:
Would it be possible to add a description on those fields too please?

Reply:
WDYM?

Other fields don't have a description here:
```yaml
Channel:
  properties:
    created_at:
      format: date-time
      type: string
    hard_capability:
      type: integer
    id:
      type: string
    label:
      type: string
    name:
      type: string
    soft_capability:
      type: integer
    source_ids:
      items:
        type: string
      type: array
    task_timeout_seconds:
      type: integer
    agent_sla:
      type: integer
    agent_sla_threshold:
      type: integer
    updated_at:
      format: date-time
      type: string
    auto_complete_category_ids:
      items:
        type: string
      type: array
    task_auto_complete_seconds:
      type: integer
```

Reply:
There are some descriptions on other fields like `Content.body_input_format`, imo it's a good practice that we should enforce.

```
        body_input_format:
          description: 'values can be: text or html.'
```

Reply:
Done in 3522ea4a5576d677e317b4068168216050f433b9.